utils/format-modules.js

```diff
@@ -26,7 +26,14 @@ function getModuleName(module) {
       .slice(0, SCOPED_PACKAGE_INDEX)
       .reduce((x, y) => x + y);
   }
-  return module.baseName.split("/")[0];
+  const nameComponents = module.baseName.split("/");
```

Comment:
Is it `"/"` on Windows? Or do we need like `os.sep`?

Reply:
Good q. Let me fire up the Windows VM and try it out.